Procfile

```diff
@@ -1 +1,2 @@
-web: node lib/main production
+web:                     node lib/main server
+expireCacheBusters: node lib/main expire-cache-busters
```

Comment:
I intentionally didn't add paging to the worker-specific endpoint. I agree they shouldn't be paging.

I see your design and I think I understand it, but I don't see how it's that much nicer than this one other than not needing a cleanup process that runs once in a while. Is it just that it's cheaper? I can see the simplicity being nice, but my current approach kinda re-uses other approaches we have accross tc, which I think is nice for brain-load-sake.